mechanics/RELEASE-SCHEDULE.md

```diff
@@ -5,25 +5,33 @@ weight: 50
 type: "docs"
 ---
 
-Knative releases every 6 weeks. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
+Knative releases quarterly. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
 
-With that said, it can be useful to have a list of when future releases will happen, so this document provides a schedule for the next 6+ months of releases.
+With that said, it can be useful to have a list of when future releases will happen, so this document provides a schedule for the next 12 months of releases.
+
+## Upcoming releases
 
 | Release | Date       | EOL        | Min K8s Version | Notes                         |
 | ------- | ---------- | ---------- | --------------- | ----------------------------- |
-| 1.8     | 2022-10-04 | 2023-04-04 | 1.23            | |
-| 1.9     | 2022-11-15 | 2023-05-16 | 1.23            | |
-| 1.10    | 2023-01-10 | 2023-06-27 | 1.24            | |
-
+| 1.8     | 2022-10-18 | 2023-05-02 | 1.23            | |
+| 1.9     | 2023-01-24 | 2023-08-01 | 1.24            | |
+| 1.10    | 2023-04-25 | 2023-10-31 | 1.25            | |
```

Comment:
I don't think we know what the min K8s version here will be. 1.25 will be the min k8s version when k8s 1.27 is released, but there's no https://github.com/kubernetes/sig-release/blob/master/releases/release-1.27/README.md file with the schedule yet.

https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/README.md was released the week of May 3, for example, which would be _after_ April 25.

Reply:
Fixed in 5561bdd9ac744f159810d10292db49492afecea8